docs/OV_Runtime_UG/multi_device.md

```diff
@@ -59,8 +59,6 @@ The following commands are accepted by the API:
        :fragment: [MULTI_0]
 
 
-Notice that MULTI allows you to **change device priorities on the fly**. You can alter the order, exclude a device, and bring an excluded device back. Still, it does not allow adding new devices.
```

Comment:
Since we are removing this info, can we remove the following snippet too? If I'm not mistaken, it presents changing device priorities and it is not necessary anymore, right?

Reply:
Yes. I forgot to remove them. It is done now.